src/t/d10.clj

```diff
@@ -0,0 +1,85 @@
+(ns t.d10
+  (:gen-class)
+  (:require
+    [clojure.string :as cljstr]
+    [clojure.set :as cljset]
+    [clojure.core.match :refer [match]]))
+
+(def sample
+"89010123
+78121874
+87430965
+96549874
+45678903
+32019012
+01329801
+10456732")
+
+(defn parse-input
+  [input]
+  (->> (cljstr/split-lines input)
+       (map vec)
+       (map-indexed
+         (fn [y l]
+           (->> l
+           (map-indexed
+             (fn [x c] {(parse-long (str c)) #{[x y]}})
+             )
+           (apply merge-with into))))
+       (apply merge-with into)
+  ))
```

Comment:
This is like the fourth time this year already. Makes me wonder if we should make a "text to grid" library function. 🤔

Reply:
if you do it too, it could fit nicely into your library.